index.js

```diff
@@ -53,10 +53,16 @@ if (shouldDeleteTags) {
   console.log("🔖  corresponding tags also will be deleted");
 }
 
-let deletePattern = process.env.INPUT_DELETE_TAG_PATTERN || "";
-if (deletePattern) {
-  console.log(`releases containing ${deletePattern} will be targeted`);
+let deletePattern = new RegExp(process.env.INPUT_DELETE_TAG_PATTERN || "");
+if (process.env.INPUT_DELETE_TAG_PATTERN) {
```

Comment:
@brendanator Don't you need to have an empty check within this if() ?  What happens if `deletePattern` just before cannot be created (is empty or fails to be assigned as a valid variable... then how could the if() even evaluate that env as a boolean... it would be a a regex pattern from the previous `let` assignment, right?  Look at the above previous code lines to see how `const` variable assignment was used to do boolean checks and console logging.  Perhaps we can do something similar for `deletePattern`?

Think through this a bit please and improve a bit here the potential failures, then we can merge.